zulipterminal/ui_tools/views.py

```diff
@@ -368,6 +368,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             _, self.focus_index_before_search = self.log.get_focus()
             self.set_focus("header")
             self.stream_search_box.set_caption(" ")
+            self.view.controller.enter_editor_mode_with(self.stream_search_box)
```

Comment:
I understand what the code does but I suspect that the commit description isn't accurate.

Reply:
I've made some changes and can be found at #1072